### PR TITLE
Create NOTICE.txt

### PR DESCRIPTION
**Pull request by joshfree**

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+send a check or money order for US $5.00, including the product name, the open
+source component name, and version number, to:
+
+Source Code Compliance Team
+Microsoft Corporation
+One Microsoft Way
+Redmond, WA 98052
+USA
+
+Notwithstanding any other terms, you may reverse engineer this software to the
```
Comment (gilbertw) [inline NOTICE.txt]: missing space "Not withstanding..."

File: NOTICE.txt [hunk 2]
```diff
@@ -0,0 +1,33 @@
+azure-sdk-for-c
+
+NOTICES AND INFORMATION
+Do Not Translate or Localize
+
+This software incorporates material from third parties. Microsoft makes certain
+open source code available at https://3rdpartysource.microsoft.com, or you may
+send a check or money order for US $5.00, including the product name, the open
```
Comment (JeffreyRichter) [inline NOTICE.txt]: What do I get if I send $5 to this address?

Comment (gilbertw) [reply NOTICE.txt]: That was unclear to me as well.  Its verbatim text from MSFTs 4rd party source code site.  Perhaps they send you a usb drive with the cotents of the download?

FYI, this is the details from the site when you go to "compliance inquiry".

License and compliance inquiry
Microsoft products often incorporate material from third parties. Microsoft makes certain open source code available at https://3rdpartysource.microsoft.com, or you may send a check or money order for US $5.00, including the product name, the opens source component name, architecture, and version number, to:

Source Code Compliance Team
Microsoft Corporation
One Microsoft Way
Redmond, WA 98052
USA

Please contact opensource@microsoft.com with any questions or concerns.

Comment (gilbertw) [reply NOTICE.txt]: Although they don't say it on the site, the team confirmed to me.  They will burn a CD with the contents of the "[download]" link.

File: NOTICE.txt (hunk 1 above)
Comment (joshfree) [reply NOTICE.txt]: This is verbatim text from our OSS Champ; I'm just going to keep it as is

File: NOTICE.txt (hunk 2 above)
Comment (joshfree) [reply NOTICE.txt]: Following up offline

**Pull request merged**